pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -834,3 +835,32 @@ def processing_fn(tape):
                 return self.adjoint_jacobian(new_tape, starting_state, use_device_state)
 
             return processing_fn
+
+    @staticmethod
+    def get_c_interface():
+        """Returns a tuple consisting of the device name, and
+        the location to the shared object with the C/C++ device implementation.
+        """
+
+        lib_name = "liblightning_kokkos_catalyst.so"
+        package_root = os.path.dirname(__file__)
+
+        # The absolute path of the plugin shared object varies according to the installation mode.
+
+        # Wheel mode:
+        # Fixed location at the root of the project
+        wheel_mode_location = os.path.join(package_root, "..", lib_name)
```

Comment:
The path may be different on macOS. Could you add unit tests for this method in `tests/test_device.py` to check the support.

Reply:
Unit tests for this method are a great idea 💯

Reply:
Done

Reply:
Path is the same